### PR TITLE
Lp 53 article components

### LINKED ISSUE #51: Add a hover effect for the nav buttons and article img
**Is your feature request related to a problem? Please describe.**
The nav bars look really bland 

**Describe the solution you'd like**
To have the nav buttons to have an on haver effect to show what category the user will go to

### PR DESCRIPTION
**Pull request by Lap343**
## Changes
1. Refactored the regular article and expanded article into components

## Purpose
So these components could be used through out the app

Closes #51

### CODE REVIEW

**Review by BlackCubes — APPROVED** [review]
WOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Review by krferrell — APPROVED** [review]
Nice refactor! Looking goo.

**Comment by Lap343** [comment]
> Nice refactor! Looking goo.

Very goo

**Review by dannysanchez559 — APPROVED** [review]
much cleaner, very nice

File: modern-news-app/src/components/Article.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    const articleSizeCheck = () => {
+        if(ArticleSize === 1){
+            ArticleSizeFinal = OneArticle
+        } else if(ArticleSize === 2){
+            ArticleSizeFinal = OneHalfArticle
+        }else if(ArticleSize === 3){
+            ArticleSizeFinal = OneThirdArticle
+        }
+    }
```
Comment (kmausolf) [inline modern-news-app/src/components/Article.tsx:24]: Wait, you can do this?  Lol.  That's amazing.

Comment (Lap343) [reply modern-news-app/src/components/Article.tsx:24]: Raw talent

Comment (JorgeLVilla) [reply modern-news-app/src/components/Article.tsx:24]: Wizard.

**Review by JorgeLVilla — APPROVED** [review]
Nice job Luis!

File: modern-news-app/src/components/Article.tsx (hunk 1 above)
Comment (Lap343) [reply modern-news-app/src/components/Article.tsx:24]: JK surprised me too

**Pull request merged**